### PR TITLE
Expand Click to Load configuration

### PR DESCRIPTION
**Pull request by kzar**
**Asana Task:** https://app.asana.com/0/0/1203595187958182/f

**Description**

Expand Click to Load configuration

- Add an enabled flag for Click to Load entities, so that the feature
  can be enabled/disabled for individual entities simply.
- Add rule actions field for Click to Load entities, so that the rule
  actions don't need to be hard-coded in the extension code.

### CODE REVIEW

**Review by ladamski — CHANGES_REQUESTED** [review]

File: features/click-to-play.json
Comment (kzar) [reply features/click-to-play.json]: Good point, Done.

**Review by jdorweiler — DISMISSED** [review]

File: features/click-to-play.json
Comment (ladamski) [reply features/click-to-play.json]: Seems like "state" should be at the root level of the feature as well, if we follow the pattern in the template (and other features).  

The question is whether it should be "enabled" here or just in the `extension-override.json`.  The latter makes more sense to me, but in this respect we are totally inconsistent... some features have "state" defined in the respective feature file, others only in the override file.  Not sure if @SlayterDev has a strong option here, otherwise I'll leave it up to you to decide which way to go.

**Review by ladamski — DISMISSED** [review]
Approving conditional on resolving the outstanding comment (as to not block for another day).

File: features/click-to-play.json
Comment (kzar) [reply features/click-to-play.json]: Good point about overrides, I  have marked this as Draft while we figure out which parts of this should live i overrides vs the feature config file.

Comment (kzar) [reply features/click-to-play.json]: OK, I have now set up proper platform overrides so that Click to Load entities can be enabled/disabled per platform + are disabled by default (like features). Done.

**Pull request merged**
